Unknown commands cause errors

Using any unknown command on the CLI causes a Node error.
Eg:
```
➜  bin git:(master) ✗ ./web-ext unknown

module.js:341
    throw err;
    ^

Error: Cannot find module '../dist/web-ext'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/Users/sharang/Data/open-source/mozilla/web-ext/bin/web-ext:5:1)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Function.Module.runMain (module.js:447:10)
```

We should probably not do that and show help text to the user instead.

--- a/src/program.js
+++ b/src/program.js
@@ -24,6 +24,7 @@ export class Program {
       yargsInstance = yargs(argv);
     }
     this.yargs = yargsInstance;
+    this.yargs.strict();
     this.commands = {};
   }
 

